qemu/tests/block_kill_reconnect_with_remote_storage.py

```diff
@@ -0,0 +1,157 @@
+import logging
+import time
+
+from provider.nbd_image_export import QemuNBDExportImage
+from provider.blockdev_base import BlockdevBaseTest
+from virttest import error_context
+
+from avocado.utils import process
+
+
+class BlockReconnectTest(BlockdevBaseTest):
+    """
+    Block mirror with firewall test
+    """
+
+    def __init__(self, test, params, env):
+        self.net_down = False
+
+        super(BlockReconnectTest, self).__init__(test, params, env)
+
+    def prepare_test(self):
+        try:
+            self.create_local_image()
+            self.export_local_image_with_nbd()
+            super(BlockReconnectTest, self).prepare_test()
+        except Exception:
+            self.clean_images(self.net_down)
+            raise
+
+    def create_local_image(self):
+        image_params = self.params.object_params(
+            self.params.get("local_image_tag"))
+        local_image = self.source_disk_define_by_params(
+            image_params, self.params.get("local_image_tag"))
```

Comment:
options like 'local_image_tag', these options must be defined in your cfg, we can use directly by self.params['local_image_tag'], the get() method is used when we cannot make sure the option is defined or not, just fyi.

Reply:
Got it, thanks, checking other must required configrations in cfg.